src/kv/kv_types.h

```diff
@@ -467,6 +467,14 @@ namespace kv
     virtual bool is_backup() = 0;
     virtual bool can_replicate() = 0;
 
+    enum SignatureDisposition
```

Comment:
```suggestion
    enum class SignatureDisposition
```

Come to think of it, it'd be nice to have a linter thing that makes sure we only have enum classes.